.github/workflows/ci.yml

```diff
@@ -62,3 +62,12 @@ jobs:
         with:
           name: typioca ${{ steps.get_version.outputs.version }}
           files: execs/**/*
+  winget:
+    needs: release
+    runs-on: windows-latest # Action can only be run on Windows
+    steps:
+      - uses: vedantmgoyal2009/winget-releaser@v1
+        with:
+          identifier: bloznelis.typioca
+          version: ${{ github.ref_name }}
+          token: ${{ secrets.WINGET_TOKEN }}

```

Comment:
```suggestion
          token: ${{ secrets.PUBLIC_ACCESS_TOKEN }}
```

Reply:
We already have a token with `public_repo` access in this project, we use it for homebrew releases. Let's re-use it

Reply:
I have overlooked this, Thanks!